### PR TITLE
Update footer for desktop only to look like mozilla.org footer.

### PR DESCRIPTION
**Pull request by mlagace**
I had a couple of questions regarding some of the changes related to https://github.com/mozilla/kitsune/issues/3227 and the mockup below.

Here is the mockup we received for the footer:
support_footer_mockup_may2018

Here is what I implemented:
implemented_support_footer_may2018

Questions:
- For the `Contact Us` link, the current SUMO link uses `/contact/spaces/` while I changed it to `/contact/` since mozilla.org uses that as well. Is this ok?
- In the mockup, the Mozilla logo has the word *Support* above it but mozilla.org doesn't have that so should we add *Support* like in the mockup?
- Currently we set the `Contribute to this site` URL to `https://wiki.mozilla.org/Webdev/GetInvolved/`. Should we change this? since the page for `https://wiki.mozilla.org/Webdev/GetInvolved/support.mozilla.org` seems to be outdated.

### CODE REVIEW

**Comment by bensternthal** [comment]
Hey @mlagace !!

For questions about content, you should ping @dinodog for code it is @jpetto

**Comment by mlagace** [comment]
Thanks @bensternthal !! I'll ping them then.

**Comment by bensternthal** [comment]
Thanks.. jon petto is your code reviewer but won't be able to answer questions about content etc.

**Comment by Dinodog** [comment]
@mlagace I'm following up with others for the content questions. Should have some answers later today.

**Comment by mlagace** [comment]
Much appreciated @Dinodog thanks !!

File: kitsune/sumo/jinja2/base.html [hunk 1]
```diff
@@ -216,40 +216,75 @@
...
+      <ul>
+        <li>
+          <h5>
+            <a href="https://www.mozilla.org">{{ _('Mozilla') }}</a>
+          </h5>
+        </li>
+        <li><a href="https://www.mozilla.org/about/">{{ _('About') }}</a></li>
+        <li><a href="https://blog.mozilla.org/">{{ _('Blog') }}</a></li>
+        <li><a href="https://www.mozilla.org/contact/">{{ _('Contact Us') }}</a></li>
+        <li><a href="https://donate.mozilla.org/en-US/?presets=50,30,20,10&amount=30&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=usd">{{ _('Donate') }}</a></li>
```
[the remark this answers is not in the document]
Comment (mlagace) [reply kitsune/sumo/jinja2/base.html]: Yes you're right I'll change those.

File: kitsune/sumo/jinja2/base.html [hunk 2]
```diff
@@ -216,40 +216,75 @@
...
+        <li><a href="https://wiki.mozilla.org/Webdev/GetInvolved/">{{ _('Contribute to this site') }}</a></li>
```
[the remark this answers is not in the document]
Comment (mlagace) [reply kitsune/sumo/jinja2/base.html]: I can also change it to `/get-involved` if that would be better?

[the remark this answers is not in the document]
Comment (jpetto) [reply kitsune/sumo/jinja2/base.html]: I'll defer to @Dinodog here...

**Comment by Dinodog** [comment]
> For the Contact Us link, the current SUMO link uses /contact/spaces/ while I changed it to /contact/ since mozilla.org uses that as well. Is this ok?

That sounds right, thank you.

> In the mockup, the Mozilla logo has the word Support above it but mozilla.org doesn't have that so should we add Support like in the mockup?

Let's use the Mozilla logo without SUMO in the footer.

Still working on an answer for the third question.

**Comment by Dinodog** [comment]
> Currently we set the Contribute to this site URL to https://wiki.mozilla.org/Webdev/GetInvolved/. Should we change this? since the page for https://wiki.mozilla.org/Webdev/GetInvolved/support.mozilla.org seems to be outdated.

Please link the 'Contribute to this site' to here: https://kitsune.readthedocs.io/en/latest/

**Comment by mlagace** [comment]
Thanks @Dinodog! I'll make the changes.

**Comment by mlagace** [comment]
@jpetto this should be ready for another review but it looks like the CircleCI tests failed due to `django.db.utils.OperationalError: (2006, 'MySQL server has gone away')` which didn't happen in the previous test run. Thanks!

**Comment by glogiotatidis** [comment]
> @jpetto this should be ready for another review but it looks like the CircleCI tests failed due to django.db.utils.OperationalError: (2006, 'MySQL server has gone away') which didn't happen in the previous test run. Thanks!

I'll manually re-run the tests before merging, don't worry about this atm it's unrelated to your changes.

**Comment by mlagace** [comment]
Ok thanks @glogiotatidis!

**Comment by mlagace** [comment]
Thanks @jpetto !!

**Comment by jpetto** [comment]
Changes look good to me. Appears Circle needs a re-run though. 😕

**Comment by pmac** [comment]
@jpetto it's just that annoying "MySQL server has gone away" error. The tests passed, but that thing happens intermittently. We've yet to figure out why ಠ_ಠ

**Comment by glogiotatidis** [comment]
Deployed this to https://dev.sumo.moz.works. 

@pmac I suggest that we keep `dev` for `may-2018-redesign` branch for now and we push master directly to stage. Sounds ok?

Thanks all!

**Comment by pmac** [comment]
> @pmac I suggest that we keep dev for may-2018-redesign branch for now and we push master directly to stage. Sounds ok?

Good with me 👍

**Pull request merged**